.github/workflows/build.yml

```diff
@@ -44,3 +44,14 @@ jobs:
         with:
           branch: gh-pages
           folder: docs
+  test-coverage-badge:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Check out repository code
+      uses: actions/checkout@v2
+    - name: Generate test coverage
+      run: yarn test:ci
+    - name: Generating test coverage badges
+      uses: jpb06/jest-badges-action@latest
```

Comment:
🤔  This will create new commits every time, right?

There are other methods which involve setting up the tests to report coverage to a third party (e.g. [Coveralls](https://coveralls.io/)[^1]) and then use a badge they generate. IMO that's preferable to something creating a bunch of extraneous commits (especially since you can then click the badge and see the coverage report!)

[^1]: It's been a long time since I've set up a coverage badge for a repo, so while I remember this one, I'm sure others have come along that might be better. This is not a "we should use Coveralls" post

Reply:
Is there a reason that we wouldn't want to commit for this workflow?  I don't like the idea of robot commits, but I think it's a free option, and reassuring that such a new package has such high test coverage

Reply:
I just mean it's an _additional_ commit after any time there's a merge to main. Not my project so I'll leave it up to you, but I like a third party service, since (a) they're usually free for FOSS, (b) they host the actual coverage results, (c) they can show you the coverage history, and (d) they don't require an additional commit to publish a badge

Reply:
That seems like it's worth it, I say we do it.  It'd be nice to have the coverage history

Reply:
Hm, yeah I think it's better to have a clean git history. Where "clean" means mostly atomic commits and little-to-no robot commits. I'll look into using `coveralls` or some other workflow.